AllJoyn/Platform/Bridge/BridgeCore/Bridge.vcxproj

```diff
@@ -288,8 +289,10 @@
       <Optimization>MaxSpeed</Optimization>
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <IntrinsicFunctions>true</IntrinsicFunctions>
-      <PreprocessorDefinitions>QCC_OS_GROUP_WINDOWS;NDEBUG;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>BRIDGE_WIN10;QCC_OS_GROUP_WINDOWS;_SCL_SECURE_NO_WARNINGS;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
Does windows desktop not run on arm anymore?

---
In reply to: [75178911](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75178911) [](ancestors = 75178911,75178260,75177875,75174093)